subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,81 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more and more popular. Are you wondering why? Because a
+subscription model simplifies the business process and makes it very easy for both customers and the
+business. It ensures that the business retains more customers on a consistent basis and that clients
```

Comment:
Why does it make it easy? 

It's still not a good pitch. some ideas here: https://www.entrepreneur.com/article/243573